rabit/rabit/Album/PhotoEdit/PhotoEditViewController.swift

```diff
@@ -102,9 +102,16 @@ private extension PhotoEditViewController {
     func bind() {
         guard let viewModel = viewModel else { return }
 
+        let imageSize = CGSize(
+            width: view.bounds.width,
+            height: view.bounds.width
+        )
```

Comment:
이미지의 변의 길이를 나타내는 지역변수를 하나 추가해도 괜찮을 것 같아요!
```suggestion
        let imageLength = view.bounds.width
        let imageSize = CGSize(
            width: imageLength,
            height: imageLength
        )
```